packages/component/src/createLoadable.js

```diff
@@ -258,6 +260,8 @@ function createLoadable({
           const result = resolve(loadedModule, this.props, Loadable)
           this.state.result = result
           this.state.loading = false
+          console.log(
```

Comment:
So I was thinking about making loadable a little more _observable_ and letting someone from the outside collect metric from it:
- when and what it loads
- how much time it takes
- so on

From "this file" point of view it's the same console.log - a call to some function with some "event". And then you can _expect_ this call to happen in tests enabling you to create them.

Without such _observability_ you cannot really create "platform" tests, only something very smoke application-level one. Well, testing the outcome might sounds like a good idea, but at the library level "details matter"

Thoughts?

Reply:
Was thinking of having a dev build so all test code is removed in prod. But observability would be good.

Reply:
Dev build something like something achievable with fewer efforts.

Reply:
Added a commit now, something like that?